### PR TITLE
Update Prospective Organizers section

### PR DESCRIPTION
**Pull request by Mariatta**
- Switch to CalVer ``YYYY.0M.MICRO`` format.

- Reorganize the guides based on audience.

- Add roles and expectations to prospective organizer section.

- Update the new chapter request form.

- Moved items from Prospective Organizers section that
  are related to setting up meetup/ website into
  the Organizers section.

- Add links to organizer registration, password reset form, and organizer directory.

- Simplify the navigation.

- Add "Edit on GitHub" link.

### CODE REVIEW

**Comment by Mariatta** [comment]
Preview at: https://deploy-preview-55--pyladies-kit-previews.netlify.com/

File: docs/organizer/eventhosting/planning.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Types of Local Groups
+~~~~~~~~~~~~~~~~~~~~~
+
+A local group can be:
+
+#. a geographic group: for a city, region, state, or country
+#. a student group: for a university, college, or school
+#. an interest group: for any topic (e.g. PyGame, SciPy, Django)
+
+The larger the local group, the harder it is to meet in person. Large local group's
```
Comment (willingc) [inline docs/organizer/eventhosting/planning.rst]:
```suggestion
The larger the local group, the harder it is to meet in person. Large local group
```

File: docs/organizer/eventhosting/planning.rst [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+~~~~~~~~~~~~~~~~~~~~~
+
+A local group can be:
+
+#. a geographic group: for a city, region, state, or country
+#. a student group: for a university, college, or school
+#. an interest group: for any topic (e.g. PyGame, SciPy, Django)
+
+The larger the local group, the harder it is to meet in person. Large local group
+events will be mostly virtual meetups that occur online, whereas small local group's
```
Comment (willingc) [inline docs/organizer/eventhosting/planning.rst]:
```suggestion
events will be mostly virtual meetups that occur online, whereas small local group
```

File: docs/prospective/index.rst [hunk 3]
```diff
@@ -8,128 +11,80 @@ We’re excited to have people like you join our growing community and want to h
...
+Guidelines for Starting Your Own Group
+--------------------------------------
 
-Types of Local Groups
-~~~~~~~~~~~~~~~~~~~~~
+When you start your own PyLadies group, you'll be helping to build and grow local
+Python community, through meetups, or sharing info through mailing lists, twitter, etc.
 
-A local group can be:
+A PyLadies group is not meant for a one-time event or workshop. There is expectation
```
Comment (willingc) [inline docs/prospective/index.rst]:
```suggestion
A PyLadies group is not meant for a one-time event or workshop. There is an expectation
```

File: docs/prospective/index.rst [hunk 4]
```diff
@@ -8,128 +11,80 @@ We’re excited to have people like you join our growing community and want to h
...
+for ongoing community activities through yourf group, at least once or twice a year.
```
Comment (willingc) [inline docs/prospective/index.rst]:
```suggestion
for ongoing community activities through your group, at least once or twice a year.
```

File: docs/prospective/index.rst [hunk 5]
```diff
@@ -8,128 +11,80 @@ We’re excited to have people like you join our growing community and want to h
...
-#. a geographic group: for a city, region, state, or country
-#. a student group: for a university, college, or school
-#. an interest group: for any topic (e.g. PyGame, SciPy, Django)
+Ideally the organizers in your group should be women or women majority. We also
+require at least two co-organizers in your group to help share the workload
+and prevent burnout from any single organizer.
 
-The larger the local group, the harder it is to meet in person. Large local group's events will be mostly virtual meetups that occur online, whereas small local group's events tend to be in-person meetups.
+When requesting a chapter, we ask that it be local and city-based. If an existing
+chapter already exists in your city, a second one may be allowed depends on
```
Comment (willingc) [inline docs/prospective/index.rst]:
```suggestion
chapter already exists in your city, a second one may be allowed depending on
```

**Comment by Mariatta** [comment]
Thanks @willingc!

@econchick the change doesn't seem to be deployed. Do you have time to look into it? Or if you can give me access to the project in readthedocs, I can debug it myself. Thanks.

**Pull request merged**